### PR TITLE
Fix auth header conditionalization

### PR DESCRIPTION
**Pull request by perezd**
Resolves bug detailed here:
https://community.playfab.com/questions/48116/bug-in-ue4-getentitytoken-implementation.html

### CODE REVIEW

**Comment by pgilmorepf** [comment]
This is going to conflict with some independent bug fixes we're publishing today.
I'll take a look at the spirit of this PR, and try to integrate it after our latest changes.

**Comment by perezd** [comment]
Can you link me to the commits you're referring to? I'd like to make sure I understand the impact as well.

**Comment by perezd** [comment]
Any updates here? Is this still an issue?

**Comment by pgilmorepf** [comment]
Apologies for the delay. I reviewed the commit history.
I think this is the PR that I was thinking about:
https://github.com/PlayFab/UnrealMarketplacePlugin/commit/ae82664bfd614ece1e6ea8922c3751d7d3d8162d
But those template changes were deployed Dec 8:
https://github.com/PlayFab/UnrealMarketplacePlugin/commit/a232e7a00fe7473621500fdfdb661129d8059cbe

In hindsight, I see that your code didn't conflict, as the "upcoming" changes I expected were already published. Apologies.

File: template/templates/PlayFab/PlayFab_API.cpp.ejs [hunk 1]
```diff
@@ -311,12 +311,42 @@ void UPlayFab<%- api.name %>API::Activate()
     HttpRequest->SetVerb(TEXT("POST"));
 
     // Headers
-    if (useEntityToken)
-        HttpRequest->SetHeader(TEXT("X-EntityToken"), CallAuthenticationContext != nullptr ? CallAuthenticationContext->GetEntityToken() : pfSettings->getEntityToken());
-    else if (useSessionTicket)
-        HttpRequest->SetHeader(TEXT("X-Authorization"), CallAuthenticationContext != nullptr ? CallAuthenticationContext->GetClientSessionTicket() : pfSettings->getSessionTicket());
-    else if (useSecretKey)
-        HttpRequest->SetHeader(TEXT("X-SecretKey"), CallAuthenticationContext != nullptr ? CallAuthenticationContext->GetDeveloperSecretKey() : GetDefault<UPlayFabRuntimeSettings>()->DeveloperSecretKey);
+	// Only set one of the potential authentication headers.
```
Comment (pgilmorepf) [inline template/templates/PlayFab/PlayFab_API.cpp.ejs]: Please convert all tab-chars to 4-spaces.

**Review by pgilmorepf — APPROVED** [review]
Once you fix the tab characters, I'll get the team to merge this.
Thanks for your submission!

File: template/templates/PlayFab/PlayFab_API.cpp.ejs (hunk 1 above)
Comment (perezd) [reply template/templates/PlayFab/PlayFab_API.cpp.ejs]: OK should be all better, sorry about that!

**Review by toddbellMSFT — APPROVED** [review]

**Pull request merged**